tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -203,7 +203,23 @@ ruleTester.run('no-array-prototype-extensions', rule, {
       parser: require.resolve('@typescript-eslint/parser'),
     },
 
-    // TODO: handle non-Identifier property names:
+    // function definition with Ember Array initialization.
+    `
+      function myFunction(){
```

Comment:
To simplify the test cases, let's omit the surrounding function, since it won't have any impact.

Reply:
Done.